### PR TITLE
DEVELOP-780 Add dashboard panel contribution

### PR DESCRIPTION
**Pull request by jemmyw**
Lets you create a pull requests panel on a dashboard using any search query as per https://docs.github.com/en/github/searching-for-information-on-github/searching-on-github/searching-issues-and-pull-requests

- Reorganise files as per contribution name
- Add a panel type as a list of pull requests with a custom query

Screenshot_20210617_210755

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -4,12 +4,12 @@
...
-    "typeRoots": ["node_modules/@types", "node_modules/@aha-app"],
     "allowSyntheticDefaultImports": true,
     "strictNullChecks": true,
     "downlevelIteration": true,
     "module": "ES6",
     "moduleResolution": "node",
     "target": "ES6"
-  }
+  },
+  "include": [".aha-cache/aha.d.ts", "src"]
```
Comment (jemmyw) [inline tsconfig.json:14]: Update as per the types being downloaded change coming in the client

File: src/links.js [hunk 2]
```diff
@@ -1,20 +0,0 @@
-import { AuthProvider } from "@aha-app/aha-develop-react";
```
Comment (jemmyw) [inline src/links.js:1]: renamed to src/views/attribute.js

File: src/components/ExtensionRoot.js [hunk 3]
```diff
@@ -1,16 +1,17 @@
 import { AuthProvider } from "@aha-app/aha-develop-react";
```
Comment (jemmyw) [inline src/components/ExtensionRoot.js:1]: git has picked this up as a file rename, but actually this file moved to src/views/prsPage.js and the ExtensionRoot is an extraction from all the react entry points.

**Review by percyhanna — COMMENTED** [review]
This looks great. Marking for security review as well.

**Review by cornu-ammonis — APPROVED** [review]

**Pull request merged**